_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -111,18 +111,21 @@ entries:
           - title: Marketplace Merchant Portal Core feature integration
             url: /docs/marketplace/dev/feature-integration-guides/marketplace-merchant-portal-core-feature-integration.html
 
+          - title: Marketplace Cart feature integration
+            url: /docs/marketplace/dev/feature-integration-guides/marketplace-cart-feature-integration.html
+
```

Comment:
the old one still left

Reply:
ah, means link to the origin file name that was renamed - removed.